gap/profiling.gi

```diff
@@ -737,6 +738,73 @@ function(data, outfile)
     IO_Close(outstream);
 end);
 
+# Outputs JSON for consumption by coveralls
+InstallGlobalFunction(OutputCoverallsJsonCoverage,
+function(data, outfile, jobid, pathtoremove)
+    local outstream, lineinfo, prev, file, lines, md5sum, md5path, md5cmd_full;
+
+    md5path := DirectoriesSystemPrograms();
+    md5cmd_full := Filename( md5path, "md5sum" );
+    if md5cmd_full = fail then
+        Error("Could not locate 'md5sum' in your PATH");
```

Comment:
Note that `md5sum` is not available everywhere, e.g. on Mac OS X (and on might want to run this on Travis macOS workers). Granted, this is not a reason to *not* merge this PR, more something that could perhaps be improved later.

For reference, this is the output of md5sum vs md5 vs. `openssl md5`:
```
$ openssl md5 README.md
MD5(README.md)= 1efe4eda726fbf87fb4341fac29be778
$ md5 README.md
MD5 (README.md) = 1efe4eda726fbf87fb4341fac29be778
$ md5sum README.md
1efe4eda726fbf87fb4341fac29be778  README.md
```

Reply:
Oops, did not realize this was already merged. Moved this comment to https://github.com/gap-packages/profiling/issues/63